pkg/distributor/otel_test.go

```diff
@@ -705,33 +705,93 @@ func TestOTelDeltaIngestion(t *testing.T) {
 	}
 }
 
+// Extra labels to make a more realistic workload - taken from Kubernetes' embedded cAdvisor metrics.
+var extraLabels = []labels.Label{
```

Comment:
Very similar to here: https://github.com/grafana/mimir/blob/6178f379b9a5c70998d2bf1bbe6075bfc6443108/pkg/ingester/ingester_test.go#L10044-L10046